flask_rq.py

```diff
@@ -23,7 +23,8 @@
     'RQ_DEFAULT_HOST': 'localhost',
     'RQ_DEFAULT_PORT': 6379,
     'RQ_DEFAULT_PASSWORD': None,
-    'RQ_DEFAULT_DB': 0
+    'RQ_DEFAULT_DB': 0,
+    'RQ_DEFAULT_WORKER_CLS': Worker
```

Comment:
Oh I see, didn't initially realize that `FlaskRQWorker` was your addition a few weeks ago.